### PR TITLE
Add `ready` method

### PR DESCRIPTION
**Pull request by ctjhoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,26 @@ export default Component.extend({
...
+### Segment ready
+
+You can wait for segment readyness:
+
+```js
+// app/components/some-awesome-component.js
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
```
Comment (josemarluedke) [inline README.md:100]: Could you replace this example with a glimmer component using native classes?

Here is the sniped you could use:

```js
import Component from '@glimmer/component';
import { inject as service } from '@ember/service';

export default class MyComponent extends Component {
  @service segment;

  constructor() {
    super(...arguments);

    this.segment.ready(() => {
      console.log('segment is ready');
    });
  }
}
```

**Review by josemarluedke — COMMENTED** [review]
This is looking good. I have left an inline comment for the example, but otherwise, this looks good.

File: README.md (hunk 1 above)
Comment (ctjhoa) [reply README.md]: Fix in d50efa51325fda533c15efd37b5e4c7682060bc0

**Comment by josemarluedke** [comment]
Released under v4.2.0. Thank you for working on this.

**Pull request merged**